docs/userguide.md

```diff
@@ -43,39 +43,16 @@ custom image](development.md#test-your-changes).
 
 ### Create Clusters
 
-The quickest way to set up clusters for use with the federation-v2 control
-plane is to use [minikube](https://kubernetes.io/docs/getting-started-guides/minikube/).
+The federation v2 control plane can run on any v1.11 or greater Kubernetes clusters. The following is a list of
+Kubernetes environments that have been tested and are supported by the Federation v2 community:
 
-**NOTE:** You will need to use a minikube version that supports deploying a
-kubernetes cluster >= 1.11. Currently there is no released version of minikube
-that supports kube v1.11 with profiles so you'll need to either:
+####
```

Comment:
How about 

#### Cloud Providers

Reply:
@font and I discussed the topic of naming. We agreed thats it made sense to not use the term cloud providers so the list can be a catch all for deployment environments and why minikube is part of the list.

Reply:
I see, thanks @danehans , then we can remove this `####` as it has no content.

Reply:
ack